web/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "baseUrl": ".",
     "module": "esnext",
     "moduleResolution": "node",
-    "target": "es2016",
+    "target": "es2020",
```

Comment:
Should we target to the same ES version of the Console? https://github.com/openshift/console/blob/master/frontend/tsconfig.json#L8

Just to guarantee that any system that is able to run the Console is also able to run our plugins.

Reply:
We should but my build failed with `es2016`
I can try to check in deep the reason why but not sure to find a workaround

Reply:
ok, not a big problem from my side, I was just wondering if there is any policy at organizational level.